test/Microsoft.AspNet.Mvc.Core.Test/VirtualFileResultTest.cs

```diff
@@ -241,73 +267,29 @@ public void Constructor_SetsContentTypeAndParameters()
             Assert.Equal(expectedData, contents);
         }
 
-        [Theory]
```

Comment:
These were written when VIrtualFileResult did work with the specified paths. We don't do anything interesting with it now.